python_modules/libraries/dagster-dbt/dagster_dbt/asset_utils.py

```diff
@@ -299,7 +300,14 @@ def get_manifest_and_translator_from_dbt_assets(
     check.invariant(len(dbt_assets) == 1, "Exactly one dbt AssetsDefinition is required")
     dbt_assets_def = dbt_assets[0]
     metadata_by_key = dbt_assets_def.metadata_by_key or {}
-    first_asset_key = next(iter(dbt_assets_def.keys))
+    first_asset_key = next(
```

Comment:
To avoid introducing this extra code and an understanding of asset checks here, could we just use `metadata_by_key.keys()`?